### DISCUSSION TITLE
Are you dissatisfied with README?

### DISCUSSION THREAD

**Discussion by zero-plusplus** [zero-plusplus/vscode-autohotkey-debug#140] [General]
The [README](https://github.com/zero-plusplus/vscode-autohotkey-debug/blob/master/README.md) is now over 800 lines of text to describe the many features implemented.
This is a deliberate increase, based on our experience of having problems with poorly explained projects and spending a lot of time solving them.

However, I'm worried that the amount of text will be too much, which in turn will confuse the users.
So I opened this discussion to hear your dissatisfaction about README.

Please give me your opinions.

---

The process of porting the README to the wiki is almost finished, and only the final check is left.

I've rewritten many of the explanations in the port, so please let me know if there are any places that are difficult to understand or where the translation is not working properly.

**Comment by SAbboushi** [comment]
I have just reviewed the README and I do not find it unnecessarily long; I agree it may likely confuse new users as it did me, but I think the primary reason for this is the big learning curve to understand all the features.

However, the TOC hyperlinks are still not working for me, and that is a very inconvenient problem when trying to find something in the README from within vscode "Details".

**Reply by zero-plusplus** [reply]
Thanks for the comment.

> I think the primary reason for this is the big learning curve to understand all the features.

You're right.
What I consider may not be the amount of explanation, but directing the user to the right information.

> However, the TOC hyperlinks are still not working for me, and that is a very inconvenient problem when trying to find something in the README from within vscode "Details".

It's a temporary fix, but I've added a note about this issue and a link to the README of the marketplace where the link works. This will be applied in the next version I release in a few days.

---

I've been looking for ways to improve the README, and using the Github Wiki seems to be the best solution.
This allows you to separate the description of each feature into individual pages, so I can direct users to the information they need.

I've never used it before because I thought it was impossible to hide pre-release descriptions until after release, but apparently this problem can be avoided.

**Comment by SAbboushi** [comment]
> directing the user to the right information.

I agree this is most important using TOC and hyperlinks

> I've added a note about this issue and a link to the README of the marketplace where the link works

Thank you - that will be good workaround for now
